backend/core/src/shared/filter/index.ts

```diff
@@ -68,9 +69,13 @@ export function addFilters<FilterParams, FilterFieldMap>(
           const filterField = filterTypeToFieldMap[filterType as string]
 
           // Handle custom filters here, before dropping into generic filter handler
-          if (filterType == ListingFilterKeys.seniorHousing) {
-            addSeniorHousingQuery(qb, filterValue)
-            return
+          switch (filterType) {
+            case ListingFilterKeys.seniorHousing:
+              addSeniorHousingQuery(qb, filterValue)
+              return
+            case ListingFilterKeys.communityType:
+              addCommunityTypeQuery(qb, filterValue)
```

Comment:
We still need a place where those two filters can be distinguished but as discussed, we pulled that up to the front end layer rather than having the distinction in the backend. 

Frontend filters are now decoupled from the backend filter representation.